bql/semantic/convert.go

```diff
@@ -69,6 +70,13 @@ func (c ConsumedElement) Token() *lexer.Token {
 	return c.token
 }
 
+// Token returns the boxed token.
+func (c ConsumedElement) String() string {
+	b := bytes.NewBufferString("")
+	b.WriteString(fmt.Sprintf("{isSymbol=%v, symbol=%s, token=%s}", c.isSymbol, c.symbol, c.token))
+	return b.String()
+}
```

Comment:
Why not just return fmt.Sprintf?

Reply:
Here I was biased after just working on a bigger `String()` method, where it made sense to use a buffer. I have just changed it to `fmt.Sprintf` now, thank you Kloss.